### PR TITLE
added packages/pardi/pardi.1.0.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
The distributed execution mode is not yet supported,
but should come soon.

### CODE REVIEW

**Comment by mseri** [comment]
Does this need a lower bound on parany 6.0.0 ?

File: packages/pardi/pardi.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+homepage: "https://github.com/UnixJunkie/pardi"
+bug-reports: "https://github.com/UnixJunkie/pardi/issues"
+dev-repo: "git+https://github.com/UnixJunkie/pardi.git"
+license: "GPL"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {build}
+  "batteries"
+  "dolog"
+  "parany"
```
Comment (mseri) [inline packages/pardi/pardi.1.0.0/opam]:
```suggestion
  "parany" {>= "6.0.0"}
```

**Comment by UnixJunkie** [comment]
Yes, a lower bound version constraint on parany was needed.

**Comment by mseri** [comment]
The ocaml 4.03 and 4.04 versions are failing with
```
#=== ERROR while compiling pardi.1.0.0 ========================================#
# context              2.0.4 | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/UnixJunkie/pardi/archive/v1.0.0.tar.gz)
# path                 ~/.opam/4.03/.opam-switch/build/pardi.1.0.0
# command              ~/.opam/4.03/bin/dune build -p pardi -j 71
# exit-code            1
# env-file             ~/.opam/log/pardi-15-a54edf.env
# output-file          ~/.opam/log/pardi-15-a54edf.out
### output ###
# Info: creating file dune-project with this contents:
# | (lang dune 1.10)
# | (name pardi)
# 
#     ocamlopt src/.pardi.eobjs/native/sorted_queue.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlopt.opt -w -40 -g -I src/.pardi.eobjs/byte -I src/.pardi.eobjs/native -I /home/opam/.opam/4.03/lib/batteries -I /home/opam/.opam/4.03/lib/bytes -I /home/opam/.opam/4.03/lib/dolog -I /home/opam/.opam/4.03/lib/equeue -I /home/opam/.opam/4.03/lib/minicli -I /home/opam/.opam/4.03/lib/netcamlbox -I /home/opam/.opam/4.03/lib/netmulticore -I /home/opam/.opam/4.03/lib/netplex -I /home/opam/.opam/4.03/lib/netstring -I /home/opam/.opam/4.03/lib/netsys -I /home/opam/.opam/4.03/lib/num -I /home/opam/.opam/4.03/lib/ocaml/threads -I /home/opam/.opam/4.03/lib/parany -I /home/opam/.opam/4.03/lib/rpc -intf-suffix .ml -no-alias-deps -o src/.pardi.eobjs/native/sorted_queue.cmx -c -impl src/sorted_queue.ml)
# File "src/sorted_queue.ml", line 20, characters 8-19:
# Error: Unbound value Ht.find_opt
# Hint: Did you mean find_all?
```

Maybe some other lower bound is missing?

**Comment by UnixJunkie** [comment]
Yes, I added one.

**Comment by UnixJunkie** [comment]
Ok, CI tests looks all green now.

**Comment by mseri** [comment]
Thanks for the help

**Pull request merged**